ComponentSnapshotTestCase/CKComponentSnapshotTestCase.mm

```diff
@@ -8,7 +8,7 @@
  *
  */
 
-#import <ComponentSnapshotTestCase/CKComponentSnapshotTestCase.h>
+#import <ComponentKit/CKComponentSnapshotTestCase.h>
```

Comment:
Think of [subspec](https://guides.cocoapods.org/syntax/podspec.html#subspec) as an optional expansion library upon the parent spec. A subspec inherits the value of the attributes of the parent. 

You can add a specific subspec to a podfile for their project with the form:
`pod 'PARENT/SUBSPEC'`

Otherwise, Cocoapods uses the `default_subspec` defined in the Podspec. For example
`pod 'PARENT'` == `pod 'PARENT/default_subspec'`

The `default_spec`, of ComponentKit, is "Core". I defined Core to simply link the .h, .m and .mm files, recursively contained, in `./ComponentKit` and `./ComponentTextKit`. 

The podfile for WildeGuess imports the plain old:
`pod 'ComponentKit', :path => '.'`
In this case, cocoapods uses the default subspec of `ComponentKit/Core`. Therefore, the files from the other subspecs are not linked it into builds (production or debug).